src/mcode-pilot/components/TreatmentOptionsOutcomesTable/TreatmentOptionsOutcomesTable.js

```diff
@@ -198,6 +240,32 @@ export default class TreatmentOptionsOutcomesTable extends Component {
         );
     }
 
+    sortTreatments(propName) {
```

Comment:
The function name here is a little misleading because this isn't actually sorting the treatments, this gives you the comparator to use for sorting treatments. Can you rename this so it's more clear on what it's doing?